buildenv/jenkins/README.md

```diff
@@ -91,49 +96,26 @@ You can also request a Pull Request build from the Eclipse OpenJ9 repository - [
 - To trigger a SignedOffBy Check (Only applicable to the Extensions repos)
    - `Jenkins signed off by check`
 
-##### PullRequest Trigger Regexes
-Having a complicated regex in the pull request trigger is what allows us to launch exactly the right combination of builds we need without having to make several trigger comments. The following are examples of what regexes we use in the various jobs.
-
-- Openj9 Repo
-    - Compile
-        - `.*(\n)?\bjenkins\s+compile\b\s*(((all|(([a-z]+(32)?,)*<platform>(,[a-z]+(32)?)*))\s*(jdk[0-9n]+,)*jdk<version>(,jdk[0-9n]+)*)(\s+depends.*)?)(\n)?.*`
-    - Test
-        - `.*(\n)?\bjenkins\s+test\s+<level>\b\s*(((all|(([a-z]+(32)?,)*<platform>(,[a-z]+(32)?)*))\s*(jdk[0-9n]+,)*jdk<version>(,jdk[0-9n]+)*)(\s+depends.*)?)(\n)?.*`
-
-- OpenJDK Extensions repos
-    - Compile
-        - `.*(\n)?\bjenkins\s+compile\b\s*(((all|(([a-z]+(32)?,)*<platform>(,[a-z]+(32)?)*))\s*jdk<version>)(\s+depends.*)?))(\n)?.*`
-    - Test
-        - `.*(\n)?\bjenkins\s+test\s+<level>\b\s*(((all|([a-z]+(32)?,)*<platform>(,[a-z]+(32?))*)\s*jdk<version>)(\s+depends.*)?)(\n)?.*`
-
-
 ### Jenkins Pipelines
 
 In this section:
-- `<platform>` is aix_ppc-64_cmprssptrs | linux_390-64_cmprssptrs | linux_ppc-64_cmprssptrs_le | linux_x86-64 | linux_x86-64_cmprssptrs | win_x86-64_cmprssptrs | win_x86
+- `<platform>` is the full spec name eg. aix_ppc-64_cmprssptrs
```

Comment:
Isn't `ppc64_aix` instead of `aix_ppc-64_cmprssptrs`?

Reply:
Yes I mentioned in my last comment I have yet to update the README. I would ignore it for now.